packages/app/src/pages/dashboard/migrate.tsx

```diff
@@ -0,0 +1,314 @@
+			label: t('dashboard.stake.tabs.migrate.staked'),
+			value: truncateNumbers(stakedKwentaBalance, 2),
+			buttonLabel: t('dashboard.stake.tabs.migrate.unstake'),
+			onClick: handleUnstakeKwenta,
+			active: claimableBalance.lte(0) && stakedKwentaBalance.gt(0),
+		},
+		{
+			key: 'step-3',
+			copy: t('dashboard.stake.tabs.migrate.step-3-copy'),
+			label: t('dashboard.stake.tabs.migrate.staked'),
+			value: truncateNumbers(stakedKwentaBalanceV2, 2),
+			buttonLabel: kwentaStakingV2Approved ? t('dashboard.stake.tabs.migrate.stake') : 'Approve',
+			onClick: handleStakeKwentaV2,
+			active: claimableBalance.lte(0) && stakedKwentaBalance.lte(0),
+		},
+	]
+
+	const MIGRATE_CARDS: StakingCards[] = [
+		{
+			category: t('dashboard.stake.portfolio.balance.title'),
+			card: [
+				{
+					key: 'balance-liquid',
+					title: t('dashboard.stake.portfolio.balance.liquid'),
+					value: truncateNumbers(kwentaBalance, 2),
+					onClick: () => setCurrentTab(StakeTab.Staking),
+				},
+				{
+					key: 'balance-staked',
+					title: t('dashboard.stake.portfolio.balance.staked'),
+					value: truncateNumbers(stakedKwentaBalanceV2, 2),
+					onClick: () => setCurrentTab(StakeTab.Escrow),
+				},
+			],
+		},
+		{
+			category: t('dashboard.stake.portfolio.rewards.title'),
+			card: [
+				{
+					key: 'rewards-claimable',
+					title: t('dashboard.stake.portfolio.rewards.staking'),
+					value: truncateNumbers(claimableBalanceV2, 2),
+					onClick: () => setCurrentTab(StakeTab.Staking),
+				},
+				{
+					key: 'rewards-trading',
+					title: t('dashboard.stake.portfolio.rewards.trading'),
+					value: truncateNumbers(kwentaRewards, 4),
+					onClick: () => setCurrentTab(StakeTab.Staking),
+				},
+			],
+		},
+		{
+			category: t('dashboard.stake.portfolio.escrow.title-v2'),
+			card: [
+				{
+					key: 'escrow-staked',
+					title: t('dashboard.stake.portfolio.escrow.staked'),
+					value: truncateNumbers(stakedEscrowedKwentaBalanceV2, 2),
+					onClick: () => setCurrentTab(StakeTab.Escrow),
+				},
+				{
+					key: 'escrow-vestable',
+					title: t('dashboard.stake.portfolio.escrow.vestable'),
+					value: truncateNumbers(totalVestableV2, 2),
+					onClick: () => setCurrentTab(StakeTab.Escrow),
+				},
+			],
+		},
+		{
+			category: t('dashboard.stake.portfolio.escrow.title-v1'),
+			card: [
+				{
+					key: 'escrow-staked',
+					title: t('dashboard.stake.portfolio.escrow.staked'),
+					value: truncateNumbers(stakedEscrowedKwentaBalance, 2),
+					onClick: () => setCurrentTab(StakeTab.Escrow),
+				},
+				{
+					key: 'escrow-vestable',
+					title: t('dashboard.stake.portfolio.escrow.vestable'),
+					value: truncateNumbers(totalVestable, 2),
+					onClick: () => setCurrentTab(StakeTab.Escrow),
+				},
+			],
+		},
+	]
+
+	return (
+		<>
+			<Head>
+				<title>{t('dashboard-stake.page-title')}</title>
+			</Head>
+			<StakingHeading>
+				<FlexDivCol rowGap="5px">
+					<StyledHeading variant="h4">Migrate to Staking V2</StyledHeading>
```

Comment:
I don't think migrate should be it's own page but it should just be a component of the staking page if the user has some actions to perform on migration.

Reply:
Yes. This is only for preview purposes. Will update before the final release.